content/05.discussion.md

```diff
@@ -1,29 +1,48 @@
 ## Discussion
  
 
-Science journalism is the critical conduit between the academic and public spheres, and consequently shapes the public's view of science and scientists.
+Science journalism is the critical conduit between the academic and public spheres and consequently shapes the public's view of science and scientists.
 However, as observed in other forms of recognition in science, biases may shift coverage away from the known demographics within science [@doi:10.1101/2020.04.14.927251, @doi:10.1016/j.cell.2022.01.004].
 Ideally, scientific journalism is representative of academic papers.
-Though it would be best for news coverage to promote equitable representation, at a minimum quotes and citations would ideally match the regional and gender demographics of scientific academia.
+Though it would be best for news coverage to promote equitable representation, at a minimum, quotes and citations would ideally match the perceived ethnic and gender demographics of scientific academia.
```

Comment:
```suggestion
Though it would be best for news coverage to promote equitable representation, at a minimum, quotes and citations would ideally match the predicted name origin and gender demographics of scientific academia.
```